libmamba/src/core/fetch.cpp

```diff
@@ -391,7 +415,7 @@ namespace mamba
             {
                 LOG_ERROR << "Could not open file for download " << s->m_filename << ": "
                           << strerror(errno);
-                exit(1);
+                return size * nmemb + 1;
```

Comment:
This is equivalent to `(size * nmemb) + 1`, is this what you intended? (it's not clear for me what the `1` is).

Reply:
we just need to return a "size different from size * nmemb". We could always return `0` but it's possible (unlikely though) that we download an empty file with `0` bytes (the curl manual mentions this). I chose to just randomly add `1`. Maybe I should add a comment ...